Tests/BootCamp.Chapter.Tests/Lesson3Tests.cs

```diff
@@ -16,7 +16,7 @@ public class Lesson3Tests
         [InlineData(100, 10, 1)]
         public void CalculateBmi_With_ValidInput_Returns_Expected(float weightKg, float heightM, float expectedBmi)
```

Comment:
PLease amend all the changes made to the test project. You're not supposed to touch it.